eng/helix/content/runtests.cmd

```diff
@@ -30,8 +30,8 @@ echo "Installing Runtime"
 powershell.exe -NoProfile -ExecutionPolicy unrestricted -Command "[Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12; &([scriptblock]::Create((Invoke-WebRequest -useb 'https://dot.net/v1/dotnet-install.ps1'))) -Architecture %$arch% -Runtime dotnet -Version %$runtimeVersion% -InstallDir %DOTNET_ROOT%"
 
 set exit_code=0
-echo "Restore: dotnet restore RunTests\RunTests.csproj --source https://api.nuget.org/v3/index.json --ignore-failed-sources..."
-dotnet restore RunTests\RunTests.csproj --source https://api.nuget.org/v3/index.json --ignore-failed-sources
+echo "Restore: dotnet restore RunTests\RunTests.csproj --source https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet5/nuget/v3/index.json --ignore-failed-sources..."
```

Comment:
@BrennanConroy I had to switch feeds for some of the packages to restore properly on the windows arm queue, not sure why?

Reply:
Link to a failure otherwise I have no context to help

Reply:
This is an example: https://helixre107v0xdeko0k025g8.blob.core.windows.net/dotnet-aspnetcore-refs-pull-22002-merge-c05d1ba331d44ab196/Microsoft.AspNetCore.Blazor.DataAnnotations.Validation.Tests--net5.0/console.34513ba4.log?sv=2019-02-02&se=2020-06-10T05%3A28%3A53Z&sr=c&sp=rl&sig=O8AVhm09qCqFYpe0datZW6vd45YOGPZJ3kw8ZYjWNO0%3D

Reply:
That is super strange. I'm not against this change, but I would like to know why that package is trying to be resolved :D